addons/website/tests/test_res_users.py

```diff
@@ -7,6 +7,8 @@
 from odoo.exceptions import ValidationError
 from odoo.tools import mute_logger
 from odoo.service.model import retrying
+from odoo.addons.website.tools import MockRequest
+from odoo import Command
```

Comment:
Can't blame you on this one as it's already a mess but I'd have still tried to insert your line alphabetically like line 11 before line 6 and line 10 after it.
I also don't mind if you reorder the imports, it's worth doing it now rather than adding more disorganized stuff.